lib/internal/bootstrap/node.js

```diff
@@ -67,12 +67,24 @@ if (ownsProcessState) {
 process.config = JSON.parse(internalBinding('native_module').config);
 
 const rawMethods = internalBinding('process_methods');
+
 // Set up methods and events on the process object for the main thread
 if (isMainThread) {
+  // Cache the working directory to prevent lots of lookups. If the working
+  // directory is changed by `chdir`, it'll be updated.
+  let cachedCwd = rawMethods.cwd();
```

Comment:
This is not supposed to be run during bootstrap, it cannot be included in the snapshot. You could delay this until patchProcessObject in pre_execution.js or only get the value when it’s actually called for the first time.

Reply:
Already resolved. I now get the value when the function is called for the first time / when the cache is invalid.